prime-angular-frontend/src/app/shared/components/address-view/address-view.component.ts

```diff
@@ -29,6 +29,11 @@ export class AddressViewComponent implements OnInit {
    * Show the redirect icon.
    */
   @Input() public showRedirect: boolean;
+  /**
+   * @description
+   * Show the edit button at the bottom
+   */
+  @Input() public showEditRedirect: boolean;
```

Comment:
Both `showEditRedirect` and `showRedirect` enable editing so should `showEditRedirect` be named `showEditButton` instead?

Reply:
This change is not specific for this ticket. It fixes an existing bug where each component has two redirect controls: an icon and a button. For Address component, it was missing a  boolean variable for the button.